Makefile

```diff
@@ -132,16 +129,15 @@ image: .hack-operator-image .hack-prometheus-config-reloader-image
 # Generating #
 ##############
 
+.PHONY: vendor
 vendor:
```

Comment:
maybe change the target name since it's not about vendoring only anymore.

Reply:
Separated non-vendoring commands into a separate target.